src/Model/batchprocessing/BatchProcessFMAID2ROIName.py

```diff
@@ -0,0 +1,124 @@
+            "name": "rtss",
+            "sliceable": False
+        }
+    }
+
+    def __init__(self, progress_callback, interrupt_flag, patient_files):
+        """
+        Class initialiser function.
+        :param progress_callback: A signal that receives the current
+                                  progress of the loading.
+        :param interrupt_flag: A threading.Event() object that tells the
+                               function to stop loading.
+        :param patient_files: List of patient files.
+        """
+        # Call the parent class
+        super(BatchProcessFMAID2ROIName, self).__init__(progress_callback,
+                                                        interrupt_flag,
+                                                        patient_files)
+
+        # Set class variables
+        self.required_classes = ['rtss']
+        self.organ_names = {}
+        self.fma_ids = []
+        self.ready = self.load_images(patient_files, self.required_classes)
+        self.patient_dict_container = PatientDictContainer()
+
+    def start(self):
+        """
+        Goes through the steps of the ROI Name Cleaning process.
+        :return: True if successful, False if not.
+        """
+        # Stop loading
+        if self.interrupt_flag.is_set():
+            self.patient_dict_container.clear()
+            self.summary = "INTERRUPT"
+            return False
+
+        # Lookup ROI names in Organ List
+        self.progress_callback.emit(("Reading ROIs...", 40))
+        roi_names = self.find_fma_ids()
+
+        # Return false if RTSS has no ROIs
+        if not roi_names:
+            self.summary = "ROI_NO_FMA"
+            return False
+
+        # Convert FMA ID to ROI name 
+        rtss = self.patient_dict_container.dataset['rtss']
+        total = 0
+        progress = 40
+        step = 50/len(roi_names)
+        for fmaid in roi_names:
+            self.progress_callback.emit(("Renaming FMAs...", progress))
+            progress += step
+            rtss = self.rename(rtss, fmaid, self.organ_names[fmaid])
+            total += 1
+
+        rtss.save_as(self.patient_dict_container.filepaths['rtss'])
+
+        # Set the summary to be the number of ROIs modified and return
+        self.summary = "FMA_ID_" + str(total)
+        return True
+
+    def find_fma_ids(self):
+        """
+        Return a list of ROI names in the RTSS that are standard organ names.
+        :return: list of ROI names.
+        """
+        # Get organ names and FMA IDs if they have not been populated
+        if not self.fma_ids:
+            # Get standard organ names
+            with open(data_path('organName.csv'), 'r') as f:
+                csv_input = csv.reader(f)
+                header = next(f)  # Ignore the "header" of the column
+                for row in csv_input:
+                    self.fma_ids.append(row[1])
+                    self.organ_names[row[1]] = row[0]
+                f.close()
```

Comment:
You don't need to manually close a file that is opened with a context manager (with open() as f:), it will be automatically closed once the with statement is finished.

Reply:
You are right, this has been fixed